conf/machine/imx95-19x19-verdin.conf

```diff
@@ -60,7 +65,18 @@ DDR_FIRMWARE_NAME = " \
     lpddr5_imem_qb_v202311.bin \
 "
 
-IMXBOOT_VARIANT = ""
+M4_DEFAULT_IMAGE_MX95 = "imx95-verdin-evk_m7_TCM_power_mode_switch.bin"
```

Comment:
Could you extend the commit header and body as you add more than just new device trees.

Reply:
Fixed. I divided these changes into two distinct commits to improve the organization.